meson.build

```diff
@@ -0,0 +1,24 @@
+project('gpu-viewer',
+  license: 'GPL-3.0-or-later'
+)
+
+APPLICATION_ID = 'io.github.arunsivaramanneo.GPUViewer'
+
+subdir('data')
+bin_conf = configuration_data()
+bin_conf.set('prefix', get_option('prefix'))
+bin_conf.set('datadir', get_option('datadir'))
```

Comment:
```meson
bin_conf.set('datadir', get_option('prefix') / get_option('datadir'))
```

This guarantees that Meson can reliably do intelligent path joining, and pass the fully resolved absolute location of the datadir to the configured script.